release-notes/Cloud/dataminer_services_change_log_2025.md

```diff
@@ -9,6 +9,12 @@ The dataminer.services platform gets updated continuously. This change log can h
 > [!NOTE]
 > Many features on dataminer.services are dependent on DxMs. You can find the change logs for these under [DxM release notes](xref:DxM_RNs_index).
 
+### 1 October 2025 - Enhancement - Usage - STaaS usage billing
+
+From October 1st all the STaaS usage will be billed from the Organization's DataMiner credit balance on the first day of the month. This will take effect from as early as April 2024.
```

Comment:
Starting from 1 October 2025, automatic monthly billing will be implemented for STaaS usage. The expended credits for STaaS usage will be subtracted from the relevant organization's DataMiner credit balance on the first day of each month. This automatic billing will take effect retroactively, going as far back as April 2024.